src/components/Highlights.vue

```diff
@@ -1,26 +1,181 @@
 <template>
   <div class="highlights-bar">
-    <div class="betas">JSE Index:{{ jseIndex }}</div>
-    <div class="betas">{{ industryIndices[1] }}: {{ industryBetas[1] }}</div>
-    <div class="betas">{{ industryIndices[2] }}: {{ industryBetas[2] }}</div>
+    <div class="betas">
+      All Share Index (30 Days): {{ index1 }}
+      <span
+        id="allShareNumber"
```

Comment:
Is there a reason you are using an ID? Doesn't seem like it so I would get rid of it.

Reply:
Ah, that was left over from another attempt and I just forgot to remove it, don't think I even have any css for it. Thanks for the spot